pkg/vm.go

```diff
@@ -0,0 +1,113 @@
+package pkg
+
+import (
+	"fmt"
+	"net"
+	"os"
+)
+
+//go:generate zbusc -module vmd -version 0.0.1 -name manager -package stubs github.com/threefoldtech/zos/pkg+VMModule stubs/vmd_stub.go
+
+// VMNetworkInfo structure
+type VMNetworkInfo struct {
+	// Namespace name
+	Namespace string
+	// Tap device name
+	Tap string
+	// Mac address of the device
+	MAC net.HardwareAddr
+	// Address of the device in the form of cidr
+	AddressCIDR string
+	// Gateway gateway address
+	GatewayIP string
+	// Nameservers dns servers
+	Nameservers []string
+}
+
+// Disk specifies vm disk params
+type Disk struct {
```

Comment:
Can we rename this type to `VMDisk` or `VDisk` ? Something that express more that it is only related to VMs ?